testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# This feature depends on 
+# - minkvm_guests.feature which has already configured the KVM Minion to be a virtualization host.
+# - srv_sync_channels.feature which syncs SLES 15 SP4 and the client tools.
+# We also test 'Bootstrapping using the command line' in this feature
+
+@scope_salt
+@scope_salt_ssh
+@scope_virtualization
+@virthost_kvm
+Feature: Migrate Salt to bundled Salt on a nested Minion VM
+
+  Scenario: Log in as admin user in the Salt migration context
+    Given I am authorized for the "Admin" section
+
+  Scenario: Prepare a SLES KVM test VM and list it in the Salt migration context
+    Given I am on the Systems overview page of this "kvm_server"
+    When I delete default virtual network on "kvm_server"
+    And I create salt-sles virtual network on "kvm_server"
+    And I delete default virtual storage pool on "kvm_server"
+    And I create salt-sles virtual storage pool on "kvm_server"
+    And I create a sles virtual machine named "sles-salt-migration" on "kvm_server"
+    And I follow "Virtualization" in the content area
+    And I wait until I see "sles-salt-migration" text
+
+  Scenario: Start the SLES 15 KVM VM in the Salt migration context
+    Given I follow "Virtualization" in the content area
+    When I click on "Start" in row "sles-salt-migration"
+    Then I should see "sles-salt-migration" virtual machine running on "kvm_server"
+    # We need a small delay to give cloud init some time to do its magic
+    And I wait for "30" seconds
+
+  Scenario: Check configuration files on the SLES KVM VM in the Salt migration context
+    Then file "/etc/mdns.allow" should contain ".local" on "sles-salt-migration"
+    And file "/etc/mdns.allow" should contain ".tf.local" on "sles-salt-migration"
+    And file "/etc/salt/minion" should contain "server_id_use_crc: adler32" on "sles-salt-migration"
+    And file "/etc/avahi/avahi-daemon.conf" should contain "domain-name=tf.local" on "sles-salt-migration"
+    And file "/etc/nsswitch.conf" should contain "files mdns" on "sles-salt-migration"
```

Comment:
I wonder if we really need this scenario in the context of testing the salt migration, or this is just part of the infrastructure we make to prepare it.

Reply:
We need this. If those files do not contain the correct information the server cannot contact/see the Minion and vise versa. Onboarding is not possible without this either.